packages/e2e-tests/specs/plugins/custom-taxonomies.test.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	activatePlugin,
+	createNewPost,
+	deactivatePlugin,
+} from '@wordpress/e2e-test-utils';
+
+describe( 'Custom Taxonomies labels are used', () => {
+	beforeAll( async () => {
+		await activatePlugin( 'gutenberg-test-custom-taxonomies' );
+	} );
+
+	beforeEach( async () => {
+		await createNewPost();
+	} );
+
+	afterAll( async () => {
+		await deactivatePlugin( 'gutenberg-test-custom-taxonomies' );
+	} );
+
+	it( 'Ensures the new item custom taxonomy labels are respected', async () => {
+		const button = await page.$x( "//button[contains(text(), 'Model') and contains(@class, 'components-button')]" );
+		expect( button ).not.toBeFalsy();
+
+		const labelNew = await page.$x( "//label[@class='components-form-token-field__label' and contains(text(), 'Add New Model')]" );
+		expect( labelNew ).not.toBeFalsy();
+	} );
+
+	it( 'Ensures the custom taxonomy labels are respected', async () => {
```

Comment:
Any reason this is separate from the first test, it feels like something that can be included in the same test as it's a continuation, right?

Reply:
Yes, I have modified this for one single test